### ISSUE TITLE
Disambiguate the duplicate bolded captions for ‘federated learning’ graphics 

### ISSUE THREAD

**Issue by ChrisXKeroack** [intel/openfl#854]
**Describe the bug**
Disambiguate the duplicate bolded captions for ‘federated learning’ graphics 

**To Reproduce**
1.  https://openfl.readthedocs.io/en/latest/overview.html 

notice: main content uses the caption 'federated learning' for multiple graphics

**Expected behavior**
All charts / graphs / tables / graphics should have unique descriptions

**Screenshots**
![image](https://github.com/securefederatedai/openfl/assets/104644211/46049b1f-c028-423b-9887-4c76297de457)

**Desktop:**
This issue is not OS or browser specific, but for tracking purposes, testing was done on Win11, in Edge Version 114.0.1823.58 (Official build) (64-bit).

**Smartphone:**
This issue is not device, OS, or browser specific.

**Additional context**
This is an accessibility issue, please annotate this bug with the 'accessibility' label.
